Zenodo.js

```diff
@@ -0,0 +1,309 @@
+{
+	"translatorID": "a714cb93-6595-482f-b371-a4ca0be14449",
+	"label": "Zenodo",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://zenodo\\.org",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-06-12 21:58:19"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	Copyright © 2016 Philipp Zumstein
+	
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+    if (url.indexOf('/record/')>-1) {
+    	var collections = ZU.xpath(doc, '//div[contains(@class, "metadata")]//a[@class="navtrail"]');
+    	for (var i=0; i<collections.length; i++) {
+    		//Z.debug(collections[i].textContent);
+    		switch (collections[i].textContent) {
+    			case "Presentations":
+    				return "presentation";
+    			case "Software":
+    				return "computerProgram";
+    			case "Videos/Audio":
+    				return "videoRecording";//or audioRecording?
+    			case "Images":
+    				return "artwork";
+    			case "Presentations":
+    				return "presentation";
+    			case "Posters":
+    				return "";
```

Comment:
Yes, the empty string was my `TODO` string and it seems I forgot about some of them.